vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -788,6 +788,34 @@ function Get-SystemDriveDiskInfo {
     }
 }
 
+function Register-ExpandVolumeTask {
+    $toolsDir = "c:\aks-tools\expand-volume"
+    if (!(Test-Path -Path $toolsDir)) {
+        New-Item -ItemType Directory -Path $toolsDir | Out-Null
+    }
+    $taskScript = 
+@'
```

Comment:
Can we cache this script in the vhd directly?

Reply:
Discussed offline:
- cache files should have signature
- we just write it and remove it